src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/controller/GoalController.kt

```diff
@@ -32,6 +34,15 @@ class GoalController(private val service: GoalService) {
     )
   }
 
+  @PatchMapping("/{goalUuid}")
```

Comment:
Why PATCH over PUT?

Reply:
because we're not replacing a full resource representation: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH